### PR TITLE
bug fix: search results appear regardless of the page the user is on

### PR DESCRIPTION
**Pull request by Twixs**

### CODE REVIEW

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -1,2 +1,5 @@
-<app-albums-list></app-albums-list>
+<app-albums-list *ngIf="albums.items" [items]="albums.items" [info]="albums" name="Albums"></app-albums-list>
+<app-albums-list *ngIf="artists.items" [items]="artists.items" [info]="artists" name="Artists"></app-albums-list>
```
Comment (nazartsyhanyuk) [inline src/app/components/home/home.component.html]: why is your component still called "albums"?

Comment (Twixs) [reply src/app/components/home/home.component.html]: ¯\_(ツ)_/¯

File: src/app/components/grid-list/grid-list.component.ts [hunk 2]
```diff
@@ -25,15 +25,25 @@ export class GridListComponent {
...
-      return this.router.navigate([`/tracks`, item.id], navigationExtras);
+    switch (item.type) {
+      case 'album':
+        navigationExtras.queryParams.type = 'album';
+        break;
+      case 'artist':
+        navigationExtras.queryParams.type = 'artist';
+        break;
+      case 'playlist':
+        navigationExtras.queryParams.type = 'playlist';
```
Comment (nazartsyhanyuk) [inline src/app/components/grid-list/grid-list.component.ts]: What's the reason of this switch case statement?

Comment (Twixs) [reply src/app/components/grid-list/grid-list.component.ts]: In track-list component I use navigationExtras.queryParams.type to define which call should I execute (to search for artists or playlists or albums)

File: src/app/components/search/search.component.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <div class="search-container">
-  <input class="form-control mr-sm-2 shadow-none" type="search" placeholder="Search music..." aria-label="Search"
+  <input #search class="form-control mr-sm-2 shadow-none" type="search" placeholder="Search music..." aria-label="Search"
     (keyup)="searchString$.next($event.target.value)" />
   <button class="transparent_button">
```
Comment (nazartsyhanyuk) [inline src/app/components/search/search.component.html]: why this is a button if there is no click action?

File: src/app/components/grid-list/grid-list.component.ts (hunk 2 above)
Comment (nazartsyhanyuk) [reply src/app/components/grid-list/grid-list.component.ts]: 'navigationExtras.queryParams.type = item.type' - isn't that what you need?

**Pull request merged**